sb3_contrib/ars/policies.py

```diff
@@ -98,17 +102,7 @@ def __init__(
         squash_output: bool = False,
     ):
 
-        super().__init__(observation_space, action_space, squash_output=squash_output)
-
-        if isinstance(action_space, gym.spaces.Box):
-            action_dim = get_action_dim(action_space)
-            self.action_net = nn.Linear(self.features_dim, action_dim, bias=with_bias)
-            if squash_output:
-                self.action_net = nn.Sequential(self.action_net, nn.Tanh())
-        elif isinstance(action_space, gym.spaces.Discrete):
-            self.action_net = nn.Linear(self.features_dim, action_space.n, bias=with_bias)
-        else:
```

Comment:
I guess we handle that in the `supported_action_spaces` parameter in the algorithm and `get_action_dim` won't work in the case of discrete actions

Reply:
Actually it is handled in the super class.
Or do you mean, having a dedicated supported action space list that would be different from ARSPolicy?